api.c

```diff
@@ -572,10 +571,6 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
-		/* MAXIMUM WRITE SAME LENGTH */
-		val64 = htobe64(VPD_MAX_WRITE_SAME_LENGTH);
-		memcpy(&data[36], &val64, 8);
-
```

Comment:
Sorry for late, just came back from outside yesterday.

I tested this by using EXSI and sg3_utils and could work, and from the Kernel iblock/file emulator it set the limitation to 0xFFFF, the 2-Bytes from the WRITE SAME(10), and could be configurable.

BTW, what does "report 0x3c for the b0 vpd page, but not actually support WS and so the MAX WS was zero in there" mean ? Is the 0x3c value for b0 vpd page's PAGE LENGTH field is incorrect ?